From 79afd2c07962b24f76b52b19709ca991939f1156 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@iodev.co.uk>
Date: Fri, 01 Jul 2022 16:08:04 +0200
Subject: [PATCH] Fix declarations of header_size and index_size

Convert definitions in the header file into declarations, since both are
already defined in superformat.c.

This is required by GCC 10+, as it makes the -fno-common flag default,
causing the definitions to not merge.

Signed-off-by: Ismael Luceno <ismael@iodev.co.uk>
Upstream-Status: Pending
---
 src/superformat.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/superformat.h b/src/superformat.h
index b4b0687e6edf..c8f43a0c38a5 100644
--- a/src/superformat.h
+++ b/src/superformat.h
@@ -97,7 +97,7 @@ int compute_all_sequences(struct params *fd,
 void compute_track0_sequence(struct params *fd);
 int calc_skews(struct params *fd0, struct params *fd, int n);
 extern int verbosity;
-int header_size;
-int index_size;
+extern int header_size;
+extern int index_size;
 
 #endif
